query_test.go

```diff
@@ -7,11 +7,11 @@ import (
 	"testing"
 	"time"
 
-	cmp "github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp"
 )
 
 func mustParseTime(s string) time.Time {
-	t, err := time.Parse(time.RFC3339Nano, s)
+	t, err := time.Parse(time.RFC3339, s)
```

Comment:
is it ok that we're losing nanosecond precision here? I assume so but wanted to check

Reply:
Yah, it works fine for the tests.